src/MoBi.Assets/AppConstants.cs

```diff
@@ -643,6 +644,11 @@ public static string EditPath(string objectType, ObjectPath originalPath, Object
             return $"Changing {objectType} original path {originalPath} to new path {newPath}";
          }
 
+         public static string EditPathAndName(string objectType, ObjectPath originalPath, ObjectPath newPath, string originalName, string newName)
+         {
+            return $"Changing {objectType} original name from {originalName} to {newName} and original path {originalPath} to new path {newPath}";
```

Comment:
... and original path **from** {originalPath} to ... ?

Reply:
done

return $"Changing {objectType} original name from {originalName} to {newName} and original path from {originalPath} to {newPath}";